### PR TITLE
Added assigned methods. Added spec file and test_runner for testing methods in stat_tracker.

### PR DESCRIPTION
**Pull request by SamPuttman**

### CODE REVIEW

**Review by dylanmade — COMMENTED** [review]
Ohh, interesting - I did not realize we'd have to convert those symbols into integers. Will note that in the future

File: lib/stat_tracker.rb
Comment (dylanmade) [inline lib/stat_tracker.rb:1]: Ohh, interesting - I did not realize we'd have to convert those symbols into integers. Will note that in the future

**Comment by Chavezgm** [comment]
Nice work! I like what you did with the spec file

File: spec/stat_tracker_spec.rb
Comment (dylanmade) [inline spec/stat_tracker_spec.rb:1]: This is amazing, super useful. Thank you Sam

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -103,6 +147,15 @@ def self.from_csv(locations_hash)
 # teams.csv
   # count_of_teams - team_id
 
+  def count_of_teams
+    teams_total = []
+
+    @teams.each do |team|
```
Comment (Chavezgm) [inline lib/stat_tracker.rb:153]: Nice Job!

Comment (dylanmade) [reply lib/stat_tracker.rb:153]: Perfect, thanks dude

**Pull request merged**